app/src/main/java/com/kickstarter/viewmodels/CommentsViewModel.kt

```diff
@@ -291,6 +291,7 @@ interface CommentsViewModel {
                 .subscribe(this.isFetchingComments)
 
             apolloPaginate.paginatedData()?.share()
+                ?.skip(1)
```

Comment:
This will always jump the emision 1, are we sure that's the behaviour we are expecting?